<!DOCTYPE html>
<html prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article#
" lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Posts about reproducibility infrastructure (old posts, page 1) | Reproducible Science</title>
<link href="../../assets/css/all-nocdn.css" rel="stylesheet" type="text/css">
<meta name="theme-color" content="#5670d4">
<meta name="generator" content="Nikola (getnikola.com)">
<link rel="canonical" href="https://reproduciblescience.org/categories/reproducibility-infrastructure/index-1.html">
<link rel="favicon" href="favicon.ico" sizes="16x16">
<link rel="prev" href="." type="text/html">
<link rel="next" href="index-2.html" type="text/html">
<!--[if lt IE 9]><script src="../../assets/js/html5.js"></script><![endif]--><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/solid.css" integrity="sha384-TbilV5Lbhlwdyc4RuIV/JhD8NR+BfMrvz4BL5QFa2we1hQu6wvREr3v6XSRfCTRp" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/regular.css" integrity="sha384-avJt9MoJH2rB4PKRsJRHZv7yiFZn8LrnXuzvmZoD3fh1aL6aM6s0BBcnCvBe6XSD" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/brands.css" integrity="sha384-7xAnn7Zm3QC1jFjVc1A6v/toepoG3JXboQYzbM0jrPzou9OFXm/fY6Z/XiIebl/k" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/fontawesome.css" integrity="sha384-ozJwkrqb90Oa3ZNb+yKFW2lToAWYdTiF1vt8JiH5ptTGHTGcN7qdoR1F95e0kYyG" crossorigin="anonymous">
<!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-74828384-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-74828384-1');
</script>
</head>
<body>
<a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>

<!-- Menubar -->

<nav class="navbar navbar-expand-lg navbar-dark bg-primary static-top mb-4"><div class="container">
<!-- This keeps the margins nice -->
        <a class="navbar-brand" href="https://reproduciblescience.org/">
            <img src="../../logo.png" alt="Reproducible Science" id="logo" class="d-inline-block align-top"><span id="blog-title">Reproducible Science</span>
        </a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#bs-navbar" aria-controls="bs-navbar" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="bs-navbar" aria-expanded="false">
            <ul class="navbar-nav mr-auto">
<li class="nav-item">
<a href="../../who-we-are/" class="nav-link">Who We Are</a>
         </li>
<li class="nav-item">
<a href="../../contribute/" class="nav-link">Contribute</a>
         </li>
<li class="nav-item">
<a href="../../directory/" class="nav-link">Reproducible Resource Directory</a>

                
            </li>
</ul>
<ul class="navbar-nav navbar-right">
<li class="nav-item">
<a href="https://twitter.com/ReproFeed" class="nav-link"><i class="fab fa-twitter"></i> <span class="d-lg-none">Twitter</span></a>
         </li>
<li class="nav-item">
<a href="https://vida-nyu.github.io/reproducibility-news/feed.rss" class="nav-link"><i class="fas fa-rss"></i> <span class="d-lg-none">RSS</span></a>
         </li>
<li class="nav-item">
<a href="https://github.com/vida-nyu/reproducible-science" class="nav-link"><i class="fab fa-github"></i> <span class="d-lg-none">GitHub</span></a>

                
                
                
            </li>
</ul>
</div>
<!-- /.navbar-collapse -->
    </div>
<!-- /.container -->
</nav><!-- End of Menubar --><div class="container" id="content" role="main">
    <div class="body-content">
        <!--Body content-->
        
        


    <header><h1>Posts about reproducibility infrastructure (old posts, page 1)</h1>
        <div class="metadata">
            

            

        </div>
    </header><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://gatesopenresearch.org/articles/2-31/v1" class="u-url">DataPackageR: Reproducible data preprocessing, standardization and sharing using R/Bioconductor for collaborative data analysis</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-07-05T00:00:00-04:00" title="05-07-2018">05-07-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=DataPackageR:%20Reproducible%20data%20preprocessing,%20standardization%20and%20sharing%20using%20R/Bioconductor%20for%20collaborative%20data%20analysis;%20https://gatesopenresearch.org/articles/2-31/v1" onclick="ga('send', 'social', 'email', 'share', https://gatesopenresearch.org/articles/2-31/v1); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/DataPackageR-Reproducible-data-preprocessing-standardization/);text=DataPackageR:%20Reproducible%20data%20preprocessing,%20standardization%20and%20sharing%20using%20R/Bioconductor%20for%20collaborative%20data%20analysis,%20https://gatesopenresearch.org/articles/2-31/v1" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/DataPackageR-Reproducible-data-preprocessing-standardization/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>A central tenet of reproducible research is that scientific results are published along with the underlying data and software code necessary to reproduce and verify the findings. A host of tools and software have been released that facilitate such work-flows and scientific journals have increasingly demanded that code and primary data be made available with publications. There has been little practical advice on implementing reproducible research work-flows for large ’omics’ or systems biology data sets used by teams of analysts working in collaboration. In such instances it is important to ensure all analysts use the same version of a data set for their analyses. Yet, instantiating relational databases and standard operating procedures can be unwieldy, with high "startup" costs and poor adherence to procedures when they deviate substantially from an analyst’s usual work-flow. Ideally a reproducible research work-flow should fit naturally into an individual’s existing work-flow, with minimal disruption. Here, we provide an overview of how we have leveraged popular open source tools, including Bioconductor, Rmarkdown, git version control, R, and specifically R’s package system combined with a new tool DataPackageR, to implement a lightweight reproducible research work-flow for preprocessing large data sets, suitable for sharing among small-to-medium sized teams of computational scientists. Our primary contribution is the DataPackageR tool, which decouples time-consuming data processing from data analysis while leaving a traceable record of how raw data is processed into analysis-ready data sets. The software ensures packaged data objects are properly documented and performs checksum verification of these along with basic package version management, and importantly, leaves a record of data processing code in the form of package vignettes. Our group has implemented this work-flow to manage, analyze and report on pre-clinical immunological trial data from multi-center, multi-assay studies for the past three years.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://arxiv.org/pdf/1806.09525.pdf" class="u-url">How to Read a Research Compendium</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-30T00:00:00-04:00" title="30-06-2018">30-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=How%20to%20Read%20a%20Research%20Compendium;%20https://arxiv.org/pdf/1806.09525.pdf" onclick="ga('send', 'social', 'email', 'share', https://arxiv.org/pdf/1806.09525.pdf); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/How-to-Read-a-Research-Compendium/);text=How%20to%20Read%20a%20Research%20Compendium,%20https://arxiv.org/pdf/1806.09525.pdf" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/How-to-Read-a-Research-Compendium/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Researchers spend a great deal of time reading research papers. Keshav (2012) provides a three-pass method to researchers to improve their reading skills. This article extends Keshav's method for reading a research compendium. Research compendia are an increasingly used form of publication, which packages not only the research paper's text and figures, but also all data and software for better reproducibility. We introduce the existing conventions for research compendia and suggest how to utilise their shared properties in a structured reading process. Unlike the original, this article is not build upon a long history but intends to provide guidance at the outset of an emerging practice.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" class="u-url">YAMP: a containerised workflow enabling reproducibility in metagenomics research</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-06-19T00:00:00-04:00" title="19-06-2018">19-06-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=YAMP:%20a%20containerised%20workflow%20enabling%20reproducibility%20in%20metagenomics%20research;%20https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" onclick="ga('send', 'social', 'email', 'share', https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/YAMP-a-containerised-workflow-enabling-reproducibility-in-me/);text=YAMP:%20a%20containerised%20workflow%20enabling%20reproducibility%20in%20metagenomics%20research,%20https://watermark.silverchair.com/giy072.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAdIwggHOBgkqhkiG9w0BBwagggG_MIIBuwIBADCCAbQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9e5qZs5DsbcT7x6rAgEQgIIBhSLDG-1fl8D5GYdRDo-2wBE35B-JUWT1SELphgsGS_7CBSCg9GQc7B81NfOMex-RSvfwHbIBIbz5nksrXpTSpkpqjAMuzfrTNxDmLykrRHKT77Aor9BJNfACV3E2eOy1GXT6B08kbo77o85nn8G8vdSE9Qf7Dv-ACvv2bEi07bZrQ2WPC14oEFIOWKmorKXrhIcQrI7CrU3MyoypLWGhEsLh4BnTgSbs13V4yTpI7FbPur6wUMVBP81cru_Ud33rwrH4GKKADD8RYRMMwuN3RU_ZSl-XVDe2ph6RBw6KPfnA_imkXp8SRjPfsy5xieC8JJo4RqnQKsrMp895HOc3OI5nD0gyPJqWpPqqBOdDGb5hPaFpBfT_bJidP6xsIrQP9zleZAHSsEewOPfDOzCnycVPoJIhcyWCpTu3McLAu3IRny3XZKa2EtxvgRR9Tgcm7s-WHYh1MxvlabnKaboJE08SiPCr_T6CCKEkUhTovpwja1NrmhuQM5HYPY9ZRjrPd4-Nt2xG" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/YAMP-a-containerised-workflow-enabling-reproducibility-in-me/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>YAMP is a user-friendly workflow that enables the analysis of whole shotgun metagenomic data while using containerisation to ensure computational reproducibility and facilitate collaborative research. YAMP can be executed on any UNIX-like system, and offers seamless support for multiple job schedulers as well as for Amazon AWS cloud. Although YAMP has been developed to be ready-to-use by non-experts, bioinformaticians will appreciate its flexibility, modularisation, and simple customisation. The YAMP script, parameters, and documentation are available at https://github.com/alesssia/YAMP.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://arxiv.org/pdf/1805.00329.pdf" class="u-url">DeepDIVA: A Highly-Functional Python Framework for Reproducible Experiments</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-05-05T00:00:00-04:00" title="05-05-2018">05-05-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=DeepDIVA:%20A%20Highly-Functional%20Python%20Framework%20for%20Reproducible%20Experiments;%20https://arxiv.org/pdf/1805.00329.pdf" onclick="ga('send', 'social', 'email', 'share', https://arxiv.org/pdf/1805.00329.pdf); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/DeepDIVA-A-Highly-Functional-Python-Framework-for-Reproducib/);text=DeepDIVA:%20A%20Highly-Functional%20Python%20Framework%20for%20Reproducible%20Experiments,%20https://arxiv.org/pdf/1805.00329.pdf" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/DeepDIVA-A-Highly-Functional-Python-Framework-for-Reproducib/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>We introduce DeepDIVA: an infrastructure designed to enable quick and intuitive setup of reproducible experiments with a large range of useful analysis functionality. Reproducing scientific results can be a frustrating experience, not only in document image analysis but in machine learning in general. Using DeepDIVA a researcher can either reproduce a given experiment with a very limited amount of information or share their own experiments with others. Moreover, the framework offers a large range of functions, such as boilerplate code, keeping track of experiments, hyper-parameter optimization, and visualization of data and results. To demonstrate the effectiveness of this framework, this paper presents case studies in the area of handwritten document analysis where researchers benefit from the integrated functionality. DeepDIVA is implemented in Python and uses the deep learning framework PyTorch. It is completely open source, and accessible as Web Service through DIVAServices.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="http://swat.polymtl.ca/~foutsekh/docs/MSR-Aiko-Trace.pdf" class="u-url">Developer Interaction Traces backed by IDE Screen Recordings from Think aloud Sessions</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-04-10T00:00:00-04:00" title="10-04-2018">10-04-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Developer%20Interaction%20Traces%20backed%20by%20IDE%20Screen%20Recordings%20from%20Think%20aloud%20Sessions;%20http://swat.polymtl.ca/~foutsekh/docs/MSR-Aiko-Trace.pdf" onclick="ga('send', 'social', 'email', 'share', http://swat.polymtl.ca/~foutsekh/docs/MSR-Aiko-Trace.pdf); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Developer-Interaction-Traces-backed-by-IDE-Screen-Recordings/);text=Developer%20Interaction%20Traces%20backed%20by%20IDE%20Screen%20Recordings%20from%20Think%20aloud%20Sessions,%20http://swat.polymtl.ca/~foutsekh/docs/MSR-Aiko-Trace.pdf" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Developer-Interaction-Traces-backed-by-IDE-Screen-Recordings/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>There are two well-known difficulties to test and interpret methodologies for mining developer interaction traces: first, the lack of enough large datasets needed by mining or machine learning approaches to provide reliable results; and second, the lack of "ground truth" or empirical evidence that can be used to triangulate the results, or to verify their accuracy and correctness. Moreover, relying solely on interaction traces limits our ability to take into account contextual factors that can affect the applicability of mining techniques in other contexts, as well hinders our ability to fully understand the mechanics behind observed phenomena. The data presented in this paper attempts to alleviate these challenges by providing 600+ hours of developer interaction traces, from which 26+ hours are backed with video recordings of the IDE screen and developer’s comments. This data set is relevant to researchers interested in investigating program comprehension, and those who are developing techniques for interaction traces analysis and mining.</p>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="http://smarterpoland.pl/index.php/2017/12/boost-the-reproducibility-of-your-research-with-archivist/?utm_content=buffer7c1ab&amp;utm_medium=social&amp;utm_source=twitter.com&amp;utm_campaign=buffer" class="u-url">archivist: Boost the reproducibility of your research</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2018-04-01T00:00:00-04:00" title="01-04-2018">01-04-2018</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=archivist:%20Boost%20the%20reproducibility%20of%20your%20research;%20http://smarterpoland.pl/index.php/2017/12/boost-the-reproducibility-of-your-research-with-archivist/?utm_content=buffer7c1ab&amp;utm_medium=social&amp;utm_source=twitter.com&amp;utm_campaign=buffer" onclick="ga('send', 'social', 'email', 'share', http://smarterpoland.pl/index.php/2017/12/boost-the-reproducibility-of-your-research-with-archivist/?utm_content=buffer7c1ab&amp;utm_medium=social&amp;utm_source=twitter.com&amp;utm_campaign=buffer); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/archivist-Boost-the-reproducibility-of-your-research/);text=archivist:%20Boost%20the%20reproducibility%20of%20your%20research,%20http://smarterpoland.pl/index.php/2017/12/boost-the-reproducibility-of-your-research-with-archivist/?utm_content=buffer7c1ab&amp;utm_medium=social&amp;utm_source=twitter.com&amp;utm_campaign=buffer" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/archivist-Boost-the-reproducibility-of-your-research/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>The safest solution would be to store copies of every object, ever created during the data analysis. All forks, wrong paths, everything. Along with detailed information which functions with what parameters were used to generate each result. Something like the ultimate TimeMachine or GitHub for R objects. With such detailed information, every analysis would be auditable and replicable. Right now the full tracking of all created objects is not possible without deep changes in the R interpreter. The archivist is the light-weight version of such solution.</p>
    </div>
    </article><nav aria-label="Page navigation"><ul class="pagination pagination-lg justify-content-center">
<li class="page-item"><a href="." class="page-link" aria-label="Newer posts"><span aria-hidden="true">«</span></a></li>
      <li class="page-item "><a href="." class="page-link">1</a></li>
      <li class="page-item active"><a href="#" class="page-link">2 <span class="sr-only">(current)</span></a></li>
      <li class="page-item "><a href="index-2.html" class="page-link">3</a></li>
      <li class="page-item "><a href="index-3.html" class="page-link">4</a></li>
      <li class="page-item "><a href="index-4.html" class="page-link">5</a></li>
      <li class="page-item "><a href="index-5.html" class="page-link">6</a></li>
      <li class="page-item "><a href="index-6.html" class="page-link">7</a></li>
      <li class="page-item"><a href="index-2.html" class="page-link" aria-label="Older posts"><span aria-hidden="true">»</span></a></li>
  </ul></nav><!--End of body content--><footer id="footer" class="mt-4 mb-4">
            2020 <a href="mailto:vicky.steeves@nyu.edu">NYU Reproducibility WG</a> | <a href="https://github.com/vida-nyu/reproducible-science"><i class="fab fa-github"></i></a> <a href="https://twitter.com/ReproFeed"><i class="fab fa-twitter"></i></a> <a href="https://vida-nyu.github.io/reproducibility-news/feed.rss"><i class="fas fa-rss"></i></a>
            
        </footer>
</div>
</div>


        <script src="../../assets/js/all-nocdn.js"></script><!-- fancy dates --><script>
    moment.locale("en");
    fancydates(0, "DD:MM:YYYY");
    </script><!-- end fancy dates --><script>
    baguetteBox.run('div#content', {
        ignoreClass: 'islink',
        captions: function(element) {
            return element.getElementsByTagName('img')[0].alt;
    }});
    </script>
</body>
</html>
