<!DOCTYPE html>
<html prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article#
" lang="en">
<head>
<meta charset="utf-8">
<meta name="description" content="Promoting reproducibility and openness!">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Reproducible Science (old posts, page 3) | Reproducible Science</title>
<link href="../assets/css/all-nocdn.css" rel="stylesheet" type="text/css">
<meta name="theme-color" content="#5670d4">
<meta name="generator" content="Nikola (getnikola.com)">
<link rel="canonical" href="https://reproduciblescience.org/directory/index-3.html">
<link rel="favicon" href="favicon.ico" sizes="16x16">
<link rel="prev" href="index-2.html" type="text/html">
<link rel="next" href="index-4.html" type="text/html">
<!--[if lt IE 9]><script src="../assets/js/html5.js"></script><![endif]--><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/solid.css" integrity="sha384-TbilV5Lbhlwdyc4RuIV/JhD8NR+BfMrvz4BL5QFa2we1hQu6wvREr3v6XSRfCTRp" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/regular.css" integrity="sha384-avJt9MoJH2rB4PKRsJRHZv7yiFZn8LrnXuzvmZoD3fh1aL6aM6s0BBcnCvBe6XSD" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/brands.css" integrity="sha384-7xAnn7Zm3QC1jFjVc1A6v/toepoG3JXboQYzbM0jrPzou9OFXm/fY6Z/XiIebl/k" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/fontawesome.css" integrity="sha384-ozJwkrqb90Oa3ZNb+yKFW2lToAWYdTiF1vt8JiH5ptTGHTGcN7qdoR1F95e0kYyG" crossorigin="anonymous">
<!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-74828384-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-74828384-1');
</script>
</head>
<body>
<a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>

<!-- Menubar -->

<nav class="navbar navbar-expand-lg navbar-dark bg-primary static-top mb-4"><div class="container">
<!-- This keeps the margins nice -->
        <a class="navbar-brand" href="https://reproduciblescience.org/">
            <img src="../logo.png" alt="Reproducible Science" id="logo" class="d-inline-block align-top"><span id="blog-title">Reproducible Science</span>
        </a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#bs-navbar" aria-controls="bs-navbar" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="bs-navbar" aria-expanded="false">
            <ul class="navbar-nav mr-auto">
<li class="nav-item">
<a href="../who-we-are/" class="nav-link">Who We Are</a>
         </li>
<li class="nav-item">
<a href="../contribute/" class="nav-link">Contribute</a>
         </li>
<li class="nav-item">
<a href="." class="nav-link">Reproducible Resource Directory</a>

                
            </li>
</ul>
<ul class="navbar-nav navbar-right">
<li class="nav-item">
<a href="https://twitter.com/ReproFeed" class="nav-link"><i class="fab fa-twitter"></i> <span class="d-lg-none">Twitter</span></a>
         </li>
<li class="nav-item">
<a href="https://vida-nyu.github.io/reproducibility-news/feed.rss" class="nav-link"><i class="fas fa-rss"></i> <span class="d-lg-none">RSS</span></a>
         </li>
<li class="nav-item">
<a href="https://github.com/vida-nyu/reproducible-science" class="nav-link"><i class="fab fa-github"></i> <span class="d-lg-none">GitHub</span></a>

                
                
                
            </li>
</ul>
</div>
<!-- /.navbar-collapse -->
    </div>
<!-- /.container -->
</nav><!-- End of Menubar --><div class="container" id="content" role="main">
    <div class="body-content">
        <!--Body content-->
        
        

    


    

    <article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://www.sciencedirect.com/science/article/pii/S2352340919307875?via%3Dihub" class="u-url">Reproducibility dataset for a large experimental survey on word embeddings and ontology-based methods for word similarity</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2019-09-02T00:00:00-04:00" title="02-09-2019">02-09-2019</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Reproducibility%20dataset%20for%20a%20large%20experimental%20survey%20on%20word%20embeddings%20and%20ontology-based%20methods%20for%20word%20similarity;%20https://www.sciencedirect.com/science/article/pii/S2352340919307875?via%3Dihub" onclick="ga('send', 'social', 'email', 'share', https://www.sciencedirect.com/science/article/pii/S2352340919307875?via%3Dihub); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Reproducibility-dataset-for-a-large-experimental-survey-on-w/);text=Reproducibility%20dataset%20for%20a%20large%20experimental%20survey%20on%20word%20embeddings%20and%20ontology-based%20methods%20for%20word%20similarity,%20https://www.sciencedirect.com/science/article/pii/S2352340919307875?via%3Dihub" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Reproducibility-dataset-for-a-large-experimental-survey-on-w/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>This data article introduces a reproducibility dataset with the aim of allowing the exact replication of all experiments, results and data tables introduced in our companion paper (Lastra-Díaz et al., 2019), which introduces the largest experimental survey on ontology-based semantic similarity methods and Word Embeddings (WE) for word similarity reported in the literature. The implementation of all our experiments, as well as the gathering of all raw data derived from them, was based on the software implementation and evaluation of all methods in HESML library (Lastra-Díaz et al., 2017), and their subsequent recording with Reprozip (Chirigati et al., 2016). Raw data is made up by a collection of data files gathering the raw word-similarity values returned by each method for each word pair evaluated in any benchmark. Raw data files was processed by running a R-language script with the aim of computing all evaluation metrics reported in (Lastra-Díaz et al., 2019), such as Pearson and Spearman correlation, harmonic score and statistical significance p-values, as well as to generate automatically all data tables shown in our companion paper. Our dataset provides all input data files, resources and complementary software tools to reproduce from scratch all our experimental data, statistical analysis and reported data. Finally, our reproducibility dataset provides a self-contained experimentation platform which allows to run new word similarity benchmarks by setting up new experiments including other unconsidered methods or word similarity benchmarks.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  	<li class="tag"><a href="../categories/reprozip/">ReproZip</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="http://www.diva-portal.org/smash/get/diva2:1242254/FULLTEXT02.pdf" class="u-url">SciPipe: A workflow library for agile development ofcomplex and dynamic bioinformatics pipelines</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2019-09-02T00:00:00-04:00" title="02-09-2019">02-09-2019</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=SciPipe:%20A%20workflow%20library%20for%20agile%20development%20ofcomplex%20and%20dynamic%20bioinformatics%20pipelines;%20http://www.diva-portal.org/smash/get/diva2:1242254/FULLTEXT02.pdf" onclick="ga('send', 'social', 'email', 'share', http://www.diva-portal.org/smash/get/diva2:1242254/FULLTEXT02.pdf); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/SciPipe-A-workflow-library-for-agile-development-ofcomplex-a/);text=SciPipe:%20A%20workflow%20library%20for%20agile%20development%20ofcomplex%20and%20dynamic%20bioinformatics%20pipelines,%20http://www.diva-portal.org/smash/get/diva2:1242254/FULLTEXT02.pdf" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/SciPipe-A-workflow-library-for-agile-development-ofcomplex-a/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Background:The complex nature of biological data has driven the development of specialized software tools. Scientificworkflow management systems simplify the assembly of such tools into pipelines, assist with job automation, and aidreproducibility of analyses. Many contemporary workflow tools are specialized or not designed for highly complexworkflows, such as with nested loops, dynamic scheduling, and parametrization, which is common in, e.g., machinelearning.Findings:SciPipe is a workflow programming library implemented in the programming language Go, for managingcomplex and dynamic pipelines in bioinformatics, cheminformatics, and other fields. SciPipe helps in particular withworkflow constructs common in machine learning, such as extensive branching, parameter sweeps, and dynamicscheduling and parametrization of downstream tasks. SciPipe builds on flow-based programming principles to supportagile development of workflows based on a library of self-contained, reusable components. It supports running subsets ofworkflows for improved iterative development and provides a data-centric audit logging feature that saves a full audit tracefor every output file of a workflow, which can be converted to other formats such as HTML, TeX, and PDF on demand. Theutility of SciPipe is demonstrated with a machine learning pipeline, a genomics, and a transcriptomics pipeline.Conclusions:SciPipe provides a solution for agile development of complex and dynamic pipelines, especially in machinelearning, through a flexible application programming interface suitable for scientists used to programming or scripting.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducibility-infrastructure/">reproducibility infrastructure</a></li>          
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6657953/" class="u-url">Survey on Scientific Shared Resource Rigor and Reproducibility</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2019-09-02T00:00:00-04:00" title="02-09-2019">02-09-2019</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Survey%20on%20Scientific%20Shared%20Resource%20Rigor%20and%20Reproducibility;%20https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6657953/" onclick="ga('send', 'social', 'email', 'share', https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6657953/); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Survey-on-Scientific-Shared-Resource-Rigor-and-Reproducibili/);text=Survey%20on%20Scientific%20Shared%20Resource%20Rigor%20and%20Reproducibility,%20https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6657953/" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Survey-on-Scientific-Shared-Resource-Rigor-and-Reproducibili/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Shared scientific resources, also known as core facilities, support a significant portion of the research conducted at biomolecular research institutions. The Association of Biomolecular Resource Facilities (ABRF) established the Committee on Core Rigor and Reproducibility (CCoRRe) to further its mission of integrating advanced technologies, education, and communication in the operations of shared scientific resources in support of reproducible research. In order to first assess the needs of the scientific shared resource community, the CCoRRe solicited feedback from ABRF members via a survey. The purpose of the survey was to gain information on how U.S. National Institutes of Health (NIH) initiatives on advancing scientific rigor and reproducibility influenced current services and new technology development. In addition, the survey aimed to identify the challenges and opportunities related to implementation of new reporting requirements and to identify new practices and resources needed to ensure rigorous research. The results revealed a surprising unfamiliarity with the NIH guidelines. Many of the perceived challenges to the effective implementation of best practices (i.e., those designed to ensure rigor and reproducibility) were similarly noted as a challenge to effective provision of support services in a core setting. Further, most cores routinely use best practices and offer services that support rigor and reproducibility. These services include access to well-maintained instrumentation and training on experimental design and data analysis as well as data management. Feedback from this survey will enable the ABRF to build better educational resources and share critical best-practice guidelines. These resources will become important tools to the core community and the researchers they serve to impact rigor and transparency across the range of science and technology.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="http://hdl.handle.net/2142/105288" class="u-url">Foregrounding data curation to foster reproducibility of workflows and scientific data reuse</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2019-08-27T00:00:00-04:00" title="27-08-2019">27-08-2019</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Foregrounding%20data%20curation%20to%20foster%20reproducibility%20of%20workflows%20and%20scientific%20data%20reuse;%20http://hdl.handle.net/2142/105288" onclick="ga('send', 'social', 'email', 'share', http://hdl.handle.net/2142/105288); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Foregrounding-data-curation-to-foster-reproducibility-of-wor/);text=Foregrounding%20data%20curation%20to%20foster%20reproducibility%20of%20workflows%20and%20scientific%20data%20reuse,%20http://hdl.handle.net/2142/105288" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Foregrounding-data-curation-to-foster-reproducibility-of-wor/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Scientific data reuse requires careful curation and annotation of the data. Late stage curation activities foster FAIR principles which include metadata standards for making data findable, accessible, interoperable and reusable. However, in scientific domains such as biomolecular nuclear magnetic resonance spectroscopy, there is a considerable time lag (usually more than a year) between data creation and data deposition. It is simply not feasible to backfill the required metadata so long after the data has been created (anything not carefully recorded is forgotten) – curation activities must begin closer to (if not at the point of) data creation. The need for foregrounding data curation activities is well known. However, scientific disciplines which rely on complex experimental design, sophisticated instrumentation, and intricate processing workflows, require extra care. The knowledge gap investigated by this research proposal is to identify classes of important metadata which are hidden within the tacit knowledge of a scientist when constructing an experiment, hidden within the operational specifications of the scientific instrumentation, and hidden within the design / execution of processing workflows. Once these classes of hidden knowledge have been identified, it will be possible to explore mechanisms for preventing the loss of key metadata, either through automated conversion from existing metadata or through curation activities at the time of data creation. The first step of the research plan is to survey artifacts of scientific data creation. That is, (i) existing data files with accompanying metadata, (ii) workflows and scripts for data processing, and (iii) documentation for software and scientific instrumentation. The second step is to group, categorize, and classify the types of "hidden" knowledge discovered. For example, one class of hidden knowledge already uncovered is the implicit recording of data as its reciprocal rather than the value itself, as in magnetogyric versus gyromagnetic ratios. The third step is to design/propose classes of solutions for these classes of problems. For instance, reciprocals are often helped by being explicit with units of measurement. Careful design of metadata display and curation widgets can help expose and document tacit knowledge which would otherwise be lost.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://psyarxiv.com/9htca/" class="u-url">Lack of Reproducibility in Addiction Medicine</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2019-08-26T00:00:00-04:00" title="26-08-2019">26-08-2019</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=Lack%20of%20Reproducibility%20in%20Addiction%20Medicine;%20https://psyarxiv.com/9htca/" onclick="ga('send', 'social', 'email', 'share', https://psyarxiv.com/9htca/); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/Lack-of-Reproducibility-in-Addiction-Medicine/);text=Lack%20of%20Reproducibility%20in%20Addiction%20Medicine,%20https://psyarxiv.com/9htca/" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/Lack-of-Reproducibility-in-Addiction-Medicine/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Background and aims: Credible research emphasizes transparency, openness, and reproducibility. These characteristics are fundamental to promoting and maintaining research integrity. This aim of this study was to evaluate the current state of transparency and reproducibility in the field of addiction science. Design: Cross-sectional design Measurements:  This study used the National Library of Medicine catalog to search for all journals using the subject terms tag: Substance-Related Disorders [ST]. Journals were then searched via PubMed in the timeframe of January 1, 2014 to December 31, 2018 and 300 publications were randomly selected. A pilot-tested Google form containing reproducibility/transparency characteristics was used for data extraction by two investigators who performed this process in a duplicate and blinded fashion.  Findings: Slightly more than half of the publications were open access (152/293, 50.7%). Few publications had pre-registration (7/244, 2.87%), material availability (2/237, 1.23%), protocol availability (3/244 ,0.80%), data availability (28/244, 11.48%), and analysis script availability (2/244, 0.82%). Most publications provided a conflict of interest statement (221/293, 75.42%) and funding sources (268/293, 91.47%). One replication study was reported (1/244, 0.04%). Few publications were cited (64/238, 26.89%) and 0 were excluded from meta-analyses and/or systematic reviews. Conclusion: Our study found that current practices that promote transparency and reproducibility are lacking, and thus, there is much room for improvement. First, investigators should preregister studies prior to commencement. Researchers should also make the materials, data, analysis script publicly available. To foster reproducibility, individuals should remain transparent about funding sources for the project and financial conflicts of interest. Research stakeholders should work together toward improved solutions on these matters. With these protections in place, the field of addiction medicine can lead in dissemination of information necessary to treat patients.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><article class="h-entry post-text" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title"><a href="https://www.scirp.org/journal/paperinformation.aspx?paperid=94439" class="u-url">On the Index of Repeatability: Estimation and Sample Size Requirements</a></h1>
        <div class="metadata">
            <p class="dateline"><time class="published dt-published" datetime="2019-08-26T00:00:00-04:00" title="26-08-2019">26-08-2019</time></p>  
						<p><a href="mailto:?subject=I%20saw%20this%20and%20thought%20of%20you!&amp;body=On%20the%20Index%20of%20Repeatability:%20Estimation%20and%20Sample%20Size%20Requirements;%20https://www.scirp.org/journal/paperinformation.aspx?paperid=94439" onclick="ga('send', 'social', 'email', 'share', https://www.scirp.org/journal/paperinformation.aspx?paperid=94439); return true;">Email this article</a></p>
						<p><a href="https://twitter.com/share?url=/directory/directory/On-the-Index-of-Repeatability-Estimation-and-Sample-Size-Req/);text=On%20the%20Index%20of%20Repeatability:%20Estimation%20and%20Sample%20Size%20Requirements,%20https://www.scirp.org/journal/paperinformation.aspx?paperid=94439" onclick="ga('send', 'social', 'Twitter', 'tweet', /directory/directory/On-the-Index-of-Repeatability-Estimation-and-Sample-Size-Req/); return true;" target="_blank">Tweet this article</a></p> 
        </div>
    </header><div class="e-content entry-content">
    <p>Background: Repeatability is a statement on the magnitude of measurement error. When biomarkers are used for disease diagnoses, they should be measured accurately. Objectives: We derive an index of repeatability based on the ratio of two variance components. Estimation of the index is derived from the one-way Analysis of Variance table based on the one-way random effects model. We estimate the large sample variance of the estimator and assess its adequacy using bootstrap methods. An important requirement for valid estimation of repeatability is the availability of multiple observations on each subject taken by the same rater and under the same conditions. Methods: We use the delta method to derive the large sample variance of the estimate of repeatability index. The question related to the number of required repeats per subjects is answered by two methods. In first methods we estimate the number of repeats that minimizes the variance of the estimated repeatability index, and the second determine the number of repeats needed under cost-constraints. Results and Novel Contribution: The situation when the measurements do not follow Gaussian distribution will be dealt with. It is shown that the required sample size is quite sensitive to the relative cost. We illustrate the methodologies on the Serum Alanine-aminotransferase (ALT) available from hospital registry data for samples of males and females. Repeatability is higher among females in comparison to males.</p>
		<div id="index-tags">
		  <ul class="tags">
			Tagged:
		  	<li class="tag"><a href="../categories/reproducible-paper/">reproducible paper</a></li>          
		  </ul>
</div>
    </div>
    </article><nav aria-label="Page navigation"><ul class="pagination pagination-lg justify-content-center">
<li class="page-item"><a href="index-2.html" class="page-link" aria-label="Newer posts"><span aria-hidden="true">«</span></a></li>
      <li class="page-item "><a href="." class="page-link">1</a></li>
      <li class="page-item "><a href="index-1.html" class="page-link">2</a></li>
      <li class="page-item "><a href="index-2.html" class="page-link">3</a></li>
      <li class="page-item active"><a href="#" class="page-link">4 <span class="sr-only">(current)</span></a></li>
      <li class="page-item "><a href="index-4.html" class="page-link">5</a></li>
      <li class="page-item "><a href="index-5.html" class="page-link">6</a></li>
      <li class="page-item "><a href="index-6.html" class="page-link">7</a></li>
      <li class="page-item "><a href="index-7.html" class="page-link">8</a></li>
      <li class="page-item "><a href="index-8.html" class="page-link">9</a></li>
      <li class="page-item disabled"><a href="#" class="page-link" aria-label="…"><span aria-hidden="true">…</span></a></li>
      <li class="page-item "><a href="index-86.html" class="page-link">87</a></li>
      <li class="page-item"><a href="index-4.html" class="page-link" aria-label="Older posts"><span aria-hidden="true">»</span></a></li>
  </ul></nav><!--End of body content--><footer id="footer" class="mt-4 mb-4">
            2020 <a href="mailto:vicky.steeves@nyu.edu">NYU Reproducibility WG</a> | <a href="https://github.com/vida-nyu/reproducible-science"><i class="fab fa-github"></i></a> <a href="https://twitter.com/ReproFeed"><i class="fab fa-twitter"></i></a> <a href="https://vida-nyu.github.io/reproducibility-news/feed.rss"><i class="fas fa-rss"></i></a>
            
        </footer>
</div>
</div>


        <script src="../assets/js/all-nocdn.js"></script><!-- fancy dates --><script>
    moment.locale("en");
    fancydates(0, "DD:MM:YYYY");
    </script><!-- end fancy dates --><script>
    baguetteBox.run('div#content', {
        ignoreClass: 'islink',
        captions: function(element) {
            return element.getElementsByTagName('img')[0].alt;
    }});
    </script>
</body>
</html>
